tests/SimplifyTest.elm

```diff
@@ -6466,6 +6482,22 @@ a = List.singleton >> String.fromList
                             }
                             |> Review.Test.whenFixed """module A exposing (..)
 a = String.fromChar
+"""
+                        ]
+        , test "should replace x |> f |> String.toList |> String.fromList by (x |> f)" <|
+            \() ->
+                """module A exposing (..)
+a = x |> f |> String.toList |> String.fromList
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "String.toList and String.fromList cancel each other out"
+                            , details = [ "You can replace this call by the argument given to String.toList." ]
+                            , under = "String.fromList"
+                            }
+                            |> Review.Test.whenFixed """module A exposing (..)
+a = (x |> f)
```

Comment:
Not sure why parens are introduced here :thinking:

Reply:
fixed.